frontend/app/src/views/source-data/ProcessControlView.vue

```diff
@@ -157,6 +158,29 @@ export default {
       this.populateStatus();
       this.loading = false;
     },
+    FETCH_STATUS_CODES_AVAILABLE_TO_SWITCH(thiseditStatusNewItem) {
```

Comment:
is this method in capital case?

Reply:
Good catch! That function was a computed function initially and I have moved to methods. Now renamed to fetchRowStatusCodesAvailableToSwitch